LayoutTests/platform/ios-wk2/TestExpectations

```diff
@@ -2354,3 +2354,5 @@ imported/w3c/web-platform-tests/css/css-view-transitions/input-targets-root-whil
 http/wpt/opener/child-access-parent-via-windowproxy.html [ Pass ]
 http/wpt/opener/iframe-access-top-via-windowproxy.html [ Pass ]
 http/wpt/opener/parent-access-child-via-windowproxy.html [ Pass ]
+
+webkit.org/b/267796 [ Release arm64 ] imported/w3c/web-platform-tests/requestidlecallback/deadline-max-rAF-dynamic.html [ Pass Failure ]
```

Comment:
We don't need to add arm64 to this. That only corresponds to architecture running on macOS. Since this is an iOS test failure seen on a simulator architecture here doesn't matter. Please remove arm64 from this and resubmit.